taqueria-vscode-extension/src/lib/helpers.ts

```diff
@@ -619,13 +618,32 @@ export class VsCodeHelper {
 						}
 					}
 				}
+				if (!fileName) {
+					const artifactsFolder = path.join(pathToDir, config.config?.artifactsDir ?? 'artifacts');
+					const artifactsFolderUri = api.Uri.file(artifactsFolder);
+					const file = await this.vscode.window.showOpenDialog({
+						canSelectFiles: true,
+						canSelectFolders: false,
+						canSelectMany: false,
+						defaultUri: artifactsFolderUri,
+						openLabel: 'Originate',
+						title: 'Select contract to originate',
+						filters: {
+							'Contract Files': ['*.tz'],
```

Comment:
It has to exclude .*\.(default_storage|storage|parameter)

Reply:
@AlirezaHaghshenas e.g. `hello.storage.someStorageName.tz` is not a contract

Reply:
That is actually a separate task. I can exclude all storage files both from this dialog, and from the artifacts treeview.
@hu3man.